include/aie/Dialect/AIE/IR/AIETargetModel.h

```diff
@@ -635,6 +640,32 @@ class NPU2TargetModel : public BaseNPUTargetModel {
   getBurstEncodingsAndLengths() const override;
 };
 
+// A sub-portion of the Strix NPU
+class VirtualizedNPU2TargetModel : public BaseNPUTargetModel {
+  int cols;
+
+public:
+  VirtualizedNPU2TargetModel(int _cols)
+      : BaseNPUTargetModel(static_cast<TargetModelKind>(
+            static_cast<std::underlying_type_t<TargetModelKind>>(TK_AIE2_NPU2) +
```

Comment:
Does this line mean that the `kind` is expected to be continuous wrt `cols`, so that the system works for NPU2model1col...4col, and stops working if it jumps to 6col?

Reply:
It would seem so, although I didn't realize that just by looking at this code.